presto-native-execution/presto_cpp/main/tests/PrestoToVeloxQueryConfigTest.cpp

```diff
@@ -231,7 +231,7 @@ TEST_F(PrestoToVeloxQueryConfigTest, sessionPropertiesOverrideSystemConfigs) {
              try {
                expectedBytes =
                    toCapacity(expectedValue, config::CapacityUnit::BYTE);
-             } catch (const VeloxUserError& e) {
+             } catch (const VeloxUserError&) {
```

Comment:
@amitkdutta : Thanks for the cleanup.

I have generally folks use `const VeloxUserError& /*e*/` to remove unused variables... Maybe newer compilers support the syntax you are suggesting.

Don't see the styleguide give preference, but I feel the comment style is more readable.

wdyt ?

Reply:
Yea comment style is better. Let me try it in this PR. For Meta internal case, we can't build with unused variables, and this creates merge conflicts as the internal version has the unsued stuff removed.